pkg/distributor/otel.go

```diff
@@ -53,7 +54,7 @@ type OTLPHandlerLimits interface {
 
 // OTLPHandler is an http.Handler accepting OTLP write requests.
 func OTLPHandler(
-	maxRecvMsgSize int,
+	maxOtelDecompressedRecvMsgSize int,
```

Comment:
I don't see good reason to rename this parameter, and it increases diff.

"Otel" part is unnecessary, since we're already inside OTLPHandler

"decompressed" is implied -- it doesn't make sense to apply limit to compressed size only.

If we need to rename this, then I'd suggest "maxRequestSize", since we're dealing with "OTLP requests".

Reply:
Agree there's no point in renaming this parameter. It's implied that it's OTLP.

```suggestion
	maxRecvMsgSize int,
```

Reply:
the decompressed is to clarify that we limit on request bytes size after they decompress. I am ok to rename it, just found distributor.recv-msg-size unclear whether it is checking before decompression or after and had to look into code to make sure.

Reply:
I still think it's unnecessary to include `Decompressed` in the arg name. The original name is fine.

Reply:
I have no strong opinion on naming here, renaming is done in commit 1ba1bcb4854d016dfada97077fc1696ea23c84e3